mirrord/config/src/feature/network/outgoing.rs

```diff
@@ -70,7 +70,9 @@ pub enum OutgoingFilterConfig {
 /// See the outgoing [reference](https://metalbear.co/mirrord/docs/reference/traffic/#outgoing) for more
 /// details.
 ///
-/// The `remote` and `local` config for this feature are **mutually** exclusive.
+/// You can use either the `remote` or `local` value alone to tunnel all outgoing traffic at once,
```

Comment:
Lets change the structure a bit: the remote/local text, then a small example json, then "you can also use more fine-grained configuration", then the json we have now